benchmark/king2dcompressible/ala/ala.prm

```diff
@@ -0,0 +1,286 @@
+set End time                               = 1e20
+set Output directory                       = output
+
+subsection Discretization
+  subsection Stabilization parameters
+    set alpha                                              = 2
+    set beta                                               = 0.005 # default: 0.078
+    set cR                                                 = 0.033 # default: 0.33
+  end
+end
+
+# Then there are variables that describe the tolerance of
+# the linear solver as well as how the pressure should
+# be normalized. Here, we choose a zero average pressure
+# at the surface of the domain (for the current geometry, the
+# surface is defined as the top boundary).
+set Linear solver tolerance                = 1e-7
+set Temperature solver tolerance           = 1e-10
+
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+
+
+
+subsection Adiabatic conditions model
+  #set Model name = initial profile
+  set Model name = function
+
+  subsection Function
+    set Function constants  = Di=1.0, gamma=1.0
+    set Function expression = 0.091*exp(depth*Di); gamma/Di*(exp(depth*Di/gamma)-1); exp(depth*Di/gamma)  # T,p,rho
+    set Variable names      = depth
+  end
+
+end
+
+
+# Then come a number of sections that deal with the setup
+# of the problem to solve. The first one deals with the
+# geometry of the domain within which we want to solve.
+# The sections that follow all have the same basic setup
+# where we select the name of a particular model (here,
+# the box geometry) and then, in a further subsection,
+# set the parameters that are specific to this particular
+# model.
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 1
+    set Y extent = 1
+  end
+end
+
+
+# The next section deals with the initial conditions for the
+# temperature (there are no initial conditions for the
+# velocity variable since the velocity is assumed to always
+# be in a static equilibrium with the temperature field).
+# There are a number of models with the 'function' model
+# a generic one that allows us to enter the actual initial
+# conditions in the form of a formula that can contain
+# constants. We choose a linear temperature profile that
+# matches the boundary conditions defined below plus
+# a small perturbation:
+subsection Initial conditions
+  set Model name = function
+
+  subsection Function
+    set Variable names      = x,z
+    set Function constants  = p=0.01, L=1, pi=3.1415926536, k=1
+    set Function expression =  1* ((1.0-z/L) + p*cos(k*pi*x/L/1.0)*sin(pi*z/L)) +0.091
+  end
+end
+
+subsection Initial conditions
+  set Model name = function
+  subsection Function
+#    set Function constants  = z1=0.102367, z2=0.897633, pi=3.14159265359
+#    set Function expression = if(z<z1,0.5+0.5*(z1-z)/z1,if(z>z2,0.5*(1-z)/(1-z2),0.5)) + 0.1 * cos(x*pi) * sin(z*pi)
+#    set Variable names      = x,z
```

Comment:
as in a previous .prm file, what actually happens if you comment all of this out, but still select `function`?